_rules/device-motion-disabled-c249d5.md

```diff
@@ -0,0 +1,304 @@
+---
+id: c249d5
+name: Device motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that it is possible to disable functionality that can be operated by device motion.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgments:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # (used with `npm test` to ensure validity of code snippets)
+  - 'attr-lowercase'
+---
+
+## Applicability
+
+The rule applies to an [HTML document][] with an associated [Window object][] that has one or more registered [device orientation events][device orientation] or [device motion events][device motion].
+
+## Expectation 1
+
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, an [instrument][] is available to disable the event.
```

Comment:
Is there a need for the instrument to **not** be activated through device motion…
Currently it seems that a slider on `devicemotion`, disabled on `deviceorientation`, would be good 👿 
But then the disabling on `deviceorientation` would itself need a motion disabling thingie…
Plus, instrument is currently defined as an HTML element, so an event firing on the window object probably doesn't qualify…

🤔 this should be good…

Reply:
wow, that was twisted 😁